### PR TITLE
[1LP][RFR] Replace get_details() for VMs

### PR DESCRIPTION
**Pull request by quarckster**
Intent
=========
Continuation of replacing `get_detail()` method by calling `ParametrizedSummaryTable` in respectful views.

{{pytest: -v -k "test_soft_reboot or test_gce_preemptible_provision or test_ssa_template or test_change_cpu_ram or test_vm_migrate or test_suspend or test_create_instance"  --long-running}}

### CODE REVIEW

File: cfme/common/vm.py [hunk 1]
```diff
@@ -240,11 +239,12 @@ def ip_address(self):
 
     @property
     def is_retired(self):
-        """"Check retirement status of vm"""
-        view = self.load_details(refresh=True)
-        if view.entities.lifecycle.get_text_of('Retirement Date').lower() != 'never':
+        """Check retirement status of vm"""
+        view = navigate_to(self, "Details", use_resetter=False)
+        if view.entities.lifecycle.get_text_of('Retirement Date').lower() != 'never' != 'never':
```
Comment (mshriver) [inline cfme/common/vm.py]: extra `!= 'never'`

File: cfme/tests/infrastructure/test_vm_migrate.py [hunk 2]
```diff
@@ -39,7 +40,8 @@ def test_vm_migrate(appliance, new_vm, provider):
         test_flag: migrate, provision
     """
     # auto_test_services should exist to test migrate VM
-    vm_host = new_vm.get_detail(properties=('Relationships', 'Host'))
+    view = navigate_to(new_vm, 'Details')
+    vm_host = view.entities.summary('Relationships').get_text_of('Host')
     migrate_to = [vds.name for vds in provider.hosts if vds.name not in vm_host][0]
```
Comment (mshriver) [inline cfme/tests/infrastructure/test_vm_migrate.py]: This is hitting an IndexError in PRT, but I don't think your change is impacting it.

**Comment by mshriver** [comment]
Looks good, straightforward change that's well scoped. PRT results are pretty dirty, but a lot of the failures do _not_ look directly related to the changes here. Please cleanup what's relevant and summarize any remaining failures in description. :+1:

File: cfme/tests/cloud_infra_common/test_vm_instance_analysis.py [hunk 3]
```diff
@@ -296,15 +297,15 @@ def test_ssa_template(local_setup_provider, provider, soft_assert, vm_analysis_p
 
     # Check release and quadricon
     quadicon_os_icon = template.find_quadicon().data['os']
-    details_os_icon = template.get_detail(
-        properties=('Properties', 'Operating System'))
+    view = navigate_to(template, 'Details')
+    details_os_icon = view.entities.summary('Properties').get_text_of('Operating System')
```
[the remark this answers is not in the document]
Comment (quarckster) [reply cfme/tests/cloud_infra_common/test_vm_instance_analysis.py]: Fixed

**Comment by quarckster** [comment]
PRT failures are related to providers issues mostly with OpenStack. `IndexError` was presented before.

File: cfme/tests/infrastructure/test_vm_migrate.py (hunk 2 above)
Comment (quarckster) [reply cfme/tests/infrastructure/test_vm_migrate.py]: Correct, I can see the same error in Jenkins.

Comment (quarckster) [reply cfme/tests/infrastructure/test_vm_migrate.py]: Actually it's fixed already in #6664

**Pull request merged**